agent/internal/docker.go

```diff
@@ -287,6 +290,46 @@ func (d *dockerActor) sendPullLogs(ctx *actor.Context, r io.Reader) error {
 	return scanner.Err()
 }
 
+func (d *dockerActor) addProxy(config *dcontainer.Config) {
+	validVars := map[string]string{
+		"HTTP_PROXY": "", "HTTPS_PROXY": "",
+		"FTP_PROXY": "", "NO_PROXY": "",
```

Comment:
non-blocking: I'm in favor of supporting the lower-case variants as well, since that is the standard in go's http package (https://golang.org/src/net/http/transport.go?s=13180:13237#L404) and in docker (https://docs.docker.com/engine/reference/builder/#predefined-args).